Aztec/Classes/TextKit/TextView.swift

```diff
@@ -228,6 +228,17 @@ open class TextView: UITextView {
 
     var maximumListIndentationLevels = 7
     
+    /// The list indent style
+    /// Default is `default`, single style for each level.
+    public var listIndentStyle: TextList.IndentStyle {
```

Comment:
Nit: Since this only applies to unordered lists, we should consider making this clearer.